src/Tizen.Multimedia.Remoting/MediaController/MediaControlServer.cs

```diff
@@ -421,31 +421,6 @@ public static void SetDisplayRotation(Rotation rotation)
         {
             Native.UpdateDisplayRotaton(Handle, rotation.ToNative()).ThrowIfError("Failed to set display rotation.");
         }
-
-        /// <summary>
-        /// Sets the index of current playing media.
-        /// </summary>
-        /// <param name="index">The index of current playing media.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="index"/> is null.</exception>
-        /// <exception cref="InvalidOperationException">
-        ///     The server is not running .<br/>
-        ///     -or-<br/>
-        ///     An internal error occurs.
-        /// </exception>
-        /// <since_tizen> 5 </since_tizen>
-        [Obsolete("Please do not use! This will be deprecated. Please use SetInfoOfCurrentPlayingMedia instead.")]
```

Comment:
OK
was marked as obsolete already in API6:  https://samsung.github.io/TizenFX/API6/api/Tizen.Multimedia.Remoting.MediaControlServer.html#Tizen_Multimedia_Remoting_MediaControlServer_SetIndexOfCurrentPlayingMedia_System_String_